### PR TITLE
114

### PR DESCRIPTION
**Pull request by BlakeDraper**

### CODE REVIEW

File: src/app/interfaces/user.ts [hunk 1]
```diff
@@ -0,0 +1,2 @@
+export interface User {
+}
```
Comment (lprivette) [inline src/app/interfaces/user.ts:2]: Should we just delete this file if it's not being used? @BlakeDraper

**Comment by lprivette** [comment]
Changelog?

File: src/app/interfaces/user.ts (hunk 1 above)
Comment (BlakeDraper) [reply src/app/interfaces/user.ts:2]: Your call. I created it to be the interface for the User, but then realized the STN services use the term 'member' (ew) within the DB. Could still be a use for the the 'User' construct within the frontend context. But you're right, it's not in use right now.

**Comment by BlakeDraper** [comment]
updated.

**Review by lprivette — APPROVED** [review]

**Pull request merged**